avatars/avatars.js

```diff
@@ -1704,6 +1715,191 @@ class Avatar {
           return animationsIdleArrays[key].animation;
         }
       }; */
+
+      const _getBlendIdleWalkRun = (mirrorFactor, angleFactor, speedFactor, k, lerpFn, target) => {
+
+        //console.log(currentSpeed);
+        // Blend: idle <-> walk
+        if (currentSpeed <= 0.45) {
```

Comment:
This is not the right way to do it. We can smoothly blend on a curve over the 
instantaneous velocity, not an if statement plus a lerp to switch between walk/run modally.